api/src/routes/track.ts

```diff
@@ -0,0 +1,11 @@
+import { Router, Response, Request } from "express";
```

Comment:
What is the purpose of this ``/track`` route?

Reply:
It's the alias of `/science` route. on older clients It spams the same dummy errors on console while It's missing.